NEWS.md

```diff
@@ -1,5 +1,9 @@
 # valr 0.5.0.9000
 
+## Minor changes
+
+* `bed_sort()` now uses base R sorting with the `radix` method for increased speed. (#353)
```

Comment:
Should also mention that you changed the arg from `sort_by` to `sorted`.

Reply:
Good idea, thanks!